config/routes/project.rb

```diff
@@ -17,7 +17,13 @@
       get :edit
       post :transfer
       resources :members, only: %i[create destroy index new update]
-      resources :samples
+      resources :samples do
+        scope module: :samples do
```

Comment:
Can you update this scope to have an `as: :samples`. This way the prefix for the path includes samples.

For example:

GET -> new_namespace_project_samples_transfer
POST -> namespace_project_samples_transfer

Without it we get:

GET -> new_namespace_project_transfer
POST -> namespace_project_transfer_index

Reply:
Please see 22ecef81bdf9ffc5ec5dcc1af6b12b79f91b02bf.

Reply:
Thanks!